Tools-Override/resources.targets

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="GenerateResourcesCode" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" />
+
+  <PropertyGroup>
+    <GenerateResourceMSBuildRuntime>CurrentRuntime</GenerateResourceMSBuildRuntime>
```

Comment:
These tell the msbuild targets here: https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.Common.CurrentVersion.targets#L2885 to use the current runtime and architecture for transforming resx -> resources. By default msbuild tries to spin up an out-of-process task host for the project's runtime and architecture so the resources are produced using the same runtime/architecture that the application is running as. For our resources this doesn't matter because they are all strings.